cmake/templates/ngraphConfig.cmake.in

```diff
@@ -52,6 +52,12 @@ if(TARGET openvino::core AND NOT TARGET ngraph::ngraph)
         INTERFACE_LINK_LIBRARIES openvino::core)
 endif()
 
+if(TARGET openvino::frontend::common AND NOT TARGET ngraph::frontend_common)
+    add_library(ngraph::frontend_common INTERFACE IMPORTED)
+    set_target_properties(ngraph::frontend_common PROPERTIES
+        INTERFACE_LINK_LIBRARIES openvino::frontend::common)
+endif()
```

Comment:
why do we need common FE for end users?

Reply:
We need it so that users who don't need frontend manager could use frontends without having to link to FEM